### ISSUE TITLE
TAA causes jittering with "Advanced Post-Processing"

### ISSUE THREAD

**Issue by Mohammad9760** [godotengine/godot#89118]
### Tested versions

occurred in ver 4.2.1 [b09f793f5] .

### System information

Linux Ubuntu 21 - Godot 4.2.1 stable

### Issue description

I tried making a post processing effect that needs to read the z buffer, I followed [this tutorial](https://docs.godotengine.org/en/stable/tutorials/shaders/advanced_postprocessing.html?highlight=full%20screen#full-screen-quad) from the docs.
and using that along Temporal AA causes jittering.

https://github.com/godotengine/godot/assets/79392031/9ff759d9-bf88-4ed7-a38e-78d267bb2d6f

here's what it looks like with FXAA : 

https://github.com/godotengine/godot/assets/79392031/6986d868-84ec-4950-a128-f886dd0e376f

### Steps to reproduce

enable TAA from the project settings.
then follow the Advanced Post-Processing tutorial from the docs.

### Minimal reproduction project (MRP)

N/A

**Comment by Calinou** [comment]
This requires https://github.com/godotengine/godot/pull/77523 to be merged to be fixed, then you need to disable motion vectors on the shader that's used on the plane in front of the camera.

**Comment by Mohammad9760** [comment]
yeah...  makes sense.
should we keep this issue open till then?

**Comment by Calinou** [comment]
>  should we keep this issue open till then?

Yes, as there was no issue tracking the thing the pull request is fixing until now.
